app/templates/buyers/award.html

```diff
@@ -5,11 +5,34 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = breadcrumbs
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({
+    "items": [
+        {
+          "href": "/",
```

Comment:
@lfdebrux Is there a reason we're not using `url_for('external.index')` in all of these? I can see `/` used on the other app templates as well...

Reply:
No reason that I'm aware of

Reply:
@katstevens / @lfdebrux Should I change all of the slashes then? The slash is what I have seen used in almost all templates. Slashes would be faster and don't rely in the view's name being registered with the flask app.

Reply:
Given that it's like this throughout the other apps, it might make sense to have a separate ticket to fix them all.

The `external.index` view name should already be available for all the FE flask apps to access.